### DISCUSSION TITLE
`source` in `package.json` cannot be an array (parcel 2)

### DISCUSSION THREAD

**Discussion by ngocvantran** [parcel-bundler/parcel#4989] [General] [answered]
According to the documentation at https://v2.parceljs.org/configuration/package-json/#source, the `source` value of `package.json` can either be a string or an array, but the code at https://github.com/parcel-bundler/parcel/blob/1f5586657f1ea8660ca1f06abdd75687851ebbb2/packages/core/core/src/requests/EntryRequest.js#L90 only support `source` being a string.
Is this the expected behavior?

Steps to reproduce:
- Create a package.json file.
- Install `parcel@next`
- Update the package.json file to add `source` value which is an array.
- Run `npx parcel build .` in the same folder as `package.json` file.

Expected behavior: it works the same way as `npx parcel build entry1 entry2`.

Actual behavior: `Error: Could not find entry` error is displayed.

**Comment by mischnic** [comment, marked as answer]
> Is this the expected behavior?

I think not
